apps/predbat/predbat.py

```diff
@@ -17,6 +17,7 @@
 from datetime import datetime, timedelta
 import hashlib
 import traceback
+from environment import is_appdaemon_environment
```

Comment:
You will need to add environment.py to the file list for Predbat to download

Reply:
Thanks, sorry this has been stuck for a bit - been busy with other stuff @springfall2008